src/Core/ServiceWrapper/CLI/InstallOption.cs

```diff
@@ -0,0 +1,17 @@
+﻿using CommandLine;
+
+namespace winsw.CLI
+{
+    [Verb("install", HelpText = "Install Windows Service Wrapper")]
+    public class InstallOption : CliOption
+    {
+        [Option("configFile", Required = false, HelpText = "Specify the Configuration file")]
+        public string configFile { get; set; }
```

Comment:
Should it be generalized in a base class? Many commands will need this parameter

Reply:
Updated